Assets/Scripts/Enemies/EnemyProjectile.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemyProjectile : MonoBehaviour
+{
+    // Start is called before the first frame update
+    public  AttackData projectileAttackData;
+
+    void Start()
+    {
+        
```

Comment:
Might be helpful to add some TODO notes saying what should happen here.

Reply:
Removed, this class just holds the attack data of the projectile.